tools/scripts/make_encdata_ex.py

```diff
@@ -35,63 +47,74 @@
 
 NUM_FILES = 12
 
-numDigits = len(str(NUM_FILES))
+NUM_DIGITS = len(str(NUM_FILES))
 
-def WriteBin(binData, fileNum):
-    target_fname = str(private_dir / output_name) + f'_{fileNum:0{numDigits}}.bin'
+def Write_Binary(bin_data, fileNum):
```

Comment:
Seeing as this is a script that creates and processes the binaries I didn't see the point in naming them. Especially because the files do not coincide 1:1 with the input and would require their own names. However, I do see the value in making identifying the files easier, so I can add an order file. Would it be reasonable to generate that within this script?

Reply:
I would sooner say that you should list it as a static file, so that we can mandate its inclusion and verify its contents within the repo. Then, you can remove this binary-mapping code from your script and directly process to target file-names built from the input stems.